File: samples-java/datalayer.client.sub/app/src/test/java/datalayer/client/sub/AppTest.java
/*
 * SPDX-FileCopyrightText: Bosch Rexroth AG
 *
 * SPDX-License-Identifier: MIT
 */

 package datalayer.client.sub;

import org.junit.jupiter.api.Test;

import datalayer.client.sub.App;

import static org.junit.jupiter.api.Assertions.*;

class AppTest {
    @Test void appHasAGreeting() {
        App classUnderTest = new App();
        assertTrue(classUnderTest.setupSystem());

    }
}

File: samples-kotlin/datalayer.provider.simple/app/src/test/kotlin/datalayer/provider/simple/AppTest.kt
/*
 * SPDX-FileCopyrightText: Bosch Rexroth AG
 *
 * SPDX-License-Identifier: MIT
 */
package datalayer.provider.simple

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

class AppTest {
    @Test
    fun appCanBeInstantiated() {
        val callbackString = ProviderNodeCallbackString()
        assertNotNull(callbackString)
    }
}
